.gitignore

```diff
@@ -28,6 +28,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+sphinx_examples.rst
```

Comment:
Let us add this after the default `.gitignore`. That means adding this at the very end of the file.